### PR TITLE
Update patch level to 4.19.227

### PR DESCRIPTION
**Pull request by rushikeshjadhav**
- Fixes issue https://github.com/xcp-ng/xcp/issues/522
    - Cumulative update till 4.19.227
    - Disabled Citrix patches that were taken from upstream
    - Disabled GFS2 patches as its not support by the distro

### CODE REVIEW

File: SPECS/kernel-alt.spec [hunk 1]
```diff
@@ -59,29 +60,34 @@ Source2: SOURCES/kernel/macros.kernel
...
+#Patch12: 0001-gfs2-Always-check-the-result-of-gfs2_rbm_from_block.patch
+#Patch13: 0001-gfs2-Clean-up-out-of-bounds-check-in-gfs2_rbm_from_b.patch
+#Patch14: 0001-gfs2-Move-rs_-sizehint-rgd_gh-fields-into-the-inode.patch
+#Patch15: 0001-gfs2-Remove-unused-RGRP_RSRV_MINBYTES-definition.patch
+#Patch16: 0001-gfs2-Rename-bitmap.bi_-len-bytes.patch
+#Patch17: 0001-gfs2-Fix-some-minor-typos.patch
+#Patch18: 0001-gfs2-Fix-marking-bitmaps-non-full.patch
+#Patch19: 0001-gfs2-Remove-unnecessary-gfs2_rlist_alloc-parameter.patch
+#Patch20: 0001-gfs2-Pass-resource-group-to-rgblk_free.patch
+##
```
Comment (stormi) [inline SPECS/kernel-alt.spec]: Remove this empty line?

File: SPECS/kernel-alt.spec [hunk 2]
```diff
@@ -915,6 +1043,12 @@ fi
 %{python2_sitearch}/*
 
 %changelog
+* Wed Feb 01 2022 Rushikesh Jadhav <rushikesh7@gmail.com> - 4.19.227-1
+- Fixes issue #522
+- Cumulative update till 4.19.227
+- Disabled Citrix patches that were taken from upstream
+- Disabled GFS2 patches as its not support by the distro
```
Comment (stormi) [inline SPECS/kernel-alt.spec]: Shouldn't gvt patches be mentioned too?

File: SPECS/kernel-alt.spec (hunk 1 above)
Comment (rushikeshjadhav) [reply SPECS/kernel-alt.spec]: Done

File: SPECS/kernel-alt.spec (hunk 2 above)
Comment (rushikeshjadhav) [reply SPECS/kernel-alt.spec]: Done

**Pull request merged**